modules/rosa-delete-cluster-admins.adoc

```diff
@@ -6,7 +6,7 @@
 
 :_content-type: PROCEDURE
 [id="rosa-delete-cluster-admins_{context}"]
-= Revoking `cluster-admin` access using the `rosa` CLI
+= Revoking `cluster-admin` access using the {product-title} (ROSA) CLI (`rosa`)
```

Comment:
Title will be too long, probably just CLI here

Reply:
As I have mentioned earlier, maybe "ROSA CLI". Let me know if you have other thoughts.